docs/cloud/csc_notebooks/guide_for_teachers.md

```diff
@@ -34,7 +34,7 @@ through `Application wizard` or `Application form`-buttons.
 
 **Application template** Template provides the base features for your application. Most of the templates are based on
 container images maintained by Notebooks team. Take a look at the 
-[image sources in notebook-images repository](https://github.com/CSCfi/notebook-images/tree/master/builds). 
+[image sources in notebook-images repository](https://github.com/CSCfi/notebooks-public-images). 
```

Comment:
This should read notebooks-public images as well (like line 25)

Reply:
fixed :hammer: